src/openloco/station.h

```diff
@@ -64,7 +64,9 @@ namespace openloco
     struct station
     {
         string_id name; // 0x00
-        uint8_t pad_02[0x08 - 0x02];
+        tile_coord_t x; // 0x02
```

Comment:
I'll need to check how we have defined the types for loco but i would expect this is incorrect. tile_coord_t usually represent tile coordinates where as this is in map coordinates which is usually 32 times a tile coordinate.

Reply:
town.h uses `tile_coord_t`, but industry.h uses `coord_t`

Reply:
Town.h is incorrect. It's units are in coord_t. If you have a moment could you either correct that in this pr or make a new pr fixing that. This one should be coord_t as well.

Reply:
Yep sure